src/org/opensolaris/opengrok/configuration/RuntimeEnvironment.java

```diff
@@ -1504,7 +1512,13 @@ public void saveStatistics(OutputStream out) throws IOException {
      * @throws ParseException
      */
     public void loadStatistics() throws IOException, ParseException {
-        loadStatistics(new File(getConfiguration().getStatisticsFilePath()));
+        String statisticsFilePath = getConfiguration().getStatisticsFilePath();
+        if(statisticsFilePath != null && !statisticsFilePath.isEmpty()){
+            File statisticsFile = new File(statisticsFilePath);
+            if(statisticsFile != null && statisticsFile.exists()){
+                loadStatistics(statisticsFile);
+            }
+        }
```

Comment:
Or throw an exception if the `getStatisticsFilePath()` is `null` that's another option how to convert the NPE into IOException and logs the problem in the higher layer. And you wouldn'ť need to do anything to the original test I guess in this case.